### ISSUE TITLE
Question about classifier performance

### ISSUE THREAD

**Issue by ghost** [avdravid/medXGAN_explanations#2]
Hello avdravid, 

First of all, thank you for sharing this great work! It has helped me greatly in understanding the general framework of explaining via generative latent space with your easy-to-follow but detailed code. I am also trying to use a simlar framework for explainability in a different domain, and have some questions:

I was wondering what the performance of your pre-trained classifier was in each of the datasets. By this I mean the testset accuracy, which I couldn't find from the paper. 

The reason why I am asking this is because I would think it will be important for the classifier to have a reasonable performance in order to explain based on it. So this actually leads to my second question, which is was there some kind of standard for having an "acceptable" amount of classification performance to be used for GAN training and eventually to explaining using the latent space. In other words, I noticed you used three conv blocks as your classifier architecture and was wondering what made that architecture feel "enough" compared to adding another convblock. 

I'm asking these questions because I am not entirely sure if the classifier performance in my dataset is high enough. Again, thanks for the repo!

**Comment by avdravid** [comment]
Hi! Thanks for reaching out. You can find out the performance of the classifiers at: https://openaccess.thecvf.com/content/CVPR2022W/FaDE-TCV/papers/Dravid_medXGAN_Visual_Explanations_for_Medical_Classifiers_Through_a_Generative_Latent_CVPRW_2022_paper.pdf

The goal is not to generate ground truth explanations, but what the classifier thinks. You can explain your classifier even if it is not performing well, but the explanations probably won't align with what a human would think

**Comment by ghost** [comment]
Hello, thanks for the reply! 
Sorry I missed the classifier performance when skimming through the experiments section

Judging by the high classifier performance and what you said, I think it is safe to assume to have as high as possible accuracy for the classifier to have it align with what a human would think. Thanks again!
